spec/api/made_to_measure/messages_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'MadeToMeasure::Messages' do
+  context 'v0.1' do
```

Comment:
It just seems icky to have to constantly call /api/v0.1/ to get anything and even at that point /api/v1. 

If we need to cut a new version in the future we can 301 /api -> /api/v1 and then implement an /api/v2 etc. That's just my two bits though.